cloud/filestore/libs/vfs_fuse/vhost/fuse_virtio.c

```diff
@@ -384,6 +369,15 @@ int fuse_cancel_request(
     return 0;
 }
 
+void fuse_reply_none(fuse_req_t req)
```

Comment:
I don't feel this override is safe and if we override here then we probably need to delete the original implementation in virtiofsd/fuse_lowlevel.c.

Maybe instead of this solution we can change the call to fuse_cancel [here](https://github.com/ydb-platform/nbs/blob/main/cloud/filestore/libs/vfs_fuse/fs.cpp#L406)
and to check if this is oneway request to avoid calling the cancel since the request will be completed during the sync call during `process_request`

Reply:
we talked about it and the current approach seems ok